client/app/telecom/telephony/alias/dashboard/confirmDeleteConfiguration/confirmDeleteConfiguration.modal.controller.js

```diff
@@ -0,0 +1,28 @@
+angular.module('managerApp').controller('TelecomTelephonyAliasConfirmDeleteConfigurationCtrl',
+  class TelecomTelephonyAliasConfirmDeleteConfigurationCtrl {
+    constructor($uibModalInstance, number, voipServiceAlias) {
+      this.$uibModalInstance = $uibModalInstance;
+
+      this.number = number;
+      this.voipServiceAlias = voipServiceAlias;
+    }
+
+    confirm() {
+      const emptyType = 'empty';
+
+      this.loading = true;
+      this.voipServiceAlias.changeNumberFeatureType(this.number, emptyType).then(() => {
+        this.$uibModalInstance.close();
+      }).catch((error) => {
+        this.$uibModalInstance.dismiss(error);
+      }).finally(() => {
+        this.loading = false;
+      });
+    }
+
+    cancel() {
+      if (!this.loading) {
```

Comment:
What if the users tries to close the dialog while the loading is on ? Nothing will happen ? Why won't you close the dialog in this case ?

Reply:
If user clicks on delete, and by error, clicks outside the popup, it closes this one. What the user will understand with that behavior ? (Plus, without blocking the closing, the dashboard will not be updated with available options and descriptions)